grails-app/migrations/0.9.x/changelog-2024-07-18-1200-alter-table-location-type-make-location-type-code-non-null.groovy

```diff
@@ -0,0 +1,14 @@
+databaseChangeLog = {
+
+    changeSet(author: "ewaterman", id: "1721321391197-94", objectQuotingStrategy: "LEGACY") {
```

Comment:
basically, the ID of the changeset should correspond to the date in the same way you did in the changelog name.
So, as an example, the file name of one of the changelog files:
![image](https://github.com/user-attachments/assets/71fcd616-8e8e-4fc6-9a8e-fa745e772d07)
it's 2024-02-22-1300
and the changesets within it are:
```
<changeSet author="kchelstowski" id="220220241300-0">
<changeSet author="kchelstowski" id="220220241300-1">
```
220220241300-0 -> 22-02-2024-1300 and the "0" is a number of the changeset, so the next one has 1, and the next should be 2, and so on.

Reply:
why DDMMYYYYTTTT and not YYYMMDDTTTT like we have in the name? Anyways, I'll fix it like you said.

Reply:
![tradition](https://github.com/user-attachments/assets/92124662-f650-46e2-825a-fad91210bd0b)